app/presenters/provider/admin/backend_apis_index_presenter.rb

```diff
@@ -16,6 +16,7 @@ def initialize(current_account:, params: {})
 
   def backend_apis
     @backend_apis ||= current_account.backend_apis
+                                     .where.not(state: 'deleted')
```

Comment:
@lvillen How does it work for products and why we don't need to filter them like this?

Reply:
Good digging!

Reply:
Yes! I was looking for an explanation and your comments were really helpful! Thanks! 🙏